pkg/subctl/cmd/gather/gather.go

```diff
@@ -172,11 +164,10 @@ func gatherDataByCluster(cluster *cmd.Cluster, directory string) {
 		if ok {
 			for dataType, ok := range gatherTypeFlags {
 				if ok {
-					info.Status = cli.NewStatus()
-					info.Status.Start(fmt.Sprintf("Gathering %s %s", module, dataType))
+					info.Status.Start("Gathering %s %s", module, dataType)
```

Comment:
Since we may not `End` the status below, we need to create a new instance as before. Whether this code should be refactored to balance the Start/End I think is out-of-scope for this PR.